eng/pipelines/mgmt.yml

```diff
@@ -1,6 +1,23 @@
 trigger:
-  - master
-  
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - sdk/core/
```

Comment:
Why include sdk/core?

Reply:
Because there is core-management now if I recall correctly and I'm assuming they have a dependency on that?

Reply:
Actually looking through this now - it doesn't appear to have a dependency. So I'll remove this until they do.